azure/container-linux/kubernetes/controllers.tf

```diff
@@ -0,0 +1,330 @@
+# TODO: Add Scale Sets implementation once support exists: https://github.com/kubernetes/kubernetes/issues/43287
+
+# Discrete DNS records for each controller's private IPv4 for etcd usage
+resource "azurerm_dns_a_record" "etcds" {
```

Comment:
Why are separate etcd records needed? (aka, why not just use static IPs?)

Reply:
@colemickens These etcd records are used as SANs in the etcd TLS asset generation process.